### DISCUSSION TITLE
CWTV no longer detecting some videos

### DISCUSSION THREAD

**Discussion by Katurday4268** [aclap-dev/video-downloadhelper#1163] [Help and Q&A]
### Your question

In recent weeks, VDH has stop detecting some videos on CWTV.  I had no issue grabbing these previously.  Currently the only show it will let me grab from the site is Whose Line.
https://www.cwtv.com/shows/whose-line-is-it-anyway/

Some I have issues with now:
https://www.cwtv.com/shows/masters-of-illusion
https://www.cwtv.com/shows/penn-teller-fool-us/
https://www.cwtv.com/shows/up-for-auction/

Thanks!

### Addon & CoApp version and details

version: 9.1.0.42
channel: stable
build date: 2024-08-02
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 1457c17f7d4743d4
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\User Name",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

With Windows 11 Pro 64-bit version 23H2 build 22631.4037, Firefox 129.0.1 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20, I had no problem downloading your Masters of Illusion show.  It did take a few moments, maybe 10 or 15 seconds, for VDH to recognize the show.  But it did recognize it without the need to launch playback in the web page.

For the other 2 . . .

![#01](https://github.com/user-attachments/assets/cc2b9b17-f949-49d9-aa35-7616ef2a3f03)
![#02](https://github.com/user-attachments/assets/4b945102-f035-4eec-a057-9ec1c59f456a)

Those are never going to download in VDH.  You need to be paying attention to the thing I've pointed out in those 2 images.  It tells you the content is protected by DRM.  I have never visited this web site before.  Well, maybe I have but my doubts tell you if I have, it's left no lasting impression.  In any case, I can't say whether those shows used to permit downloading.  If you say they did, then they did.  All I can tell you is they are protected by DRM now.  That may be a new policy the web site imposed recently.  You'll have to say whether that's the case.  But anything that shows that little icon for DRM will not download.  DRM protection exists for the express purpose of preventing downloading of copyrighted material.  My opinion is there's entirely too much use of DRM these days.  But that's the way it is.

**Reply by Katurday4268** [reply]
Thanks for that info.  I went to each episode and checked them thanks to you pointing out the DRM icon. Looks like they are changing over to DRM with many of the shows now. Seems to be random.  The one below I had no issue with, but others I had issues.  Really annoying. Thanks for the help with this!

![fool us](https://github.com/user-attachments/assets/3944ad43-ed46-432f-8cc3-d7236327f189)
![no drm](https://github.com/user-attachments/assets/8eeb0fed-369c-40c1-a260-2201a3e15cce)
